polyn_elixir_client/lib/polyn/migration/migrator.ex

```diff
@@ -163,4 +174,8 @@ defmodule Polyn.Migration.Migrator do
 
     state
   end
+
+  defp otp_app do
+    Application.fetch_env!(:polyn, :otp_app)
```

Comment:
This pattern assumes that you'll only have a single OTP app that needs to configure polyn migrations, in the future, do you think it would be helpful to allow people do so something like this?

```elixir
defmodule MyApp.PolynMigrations
  use Polyn.Migration, otp_app: :my_app
end
```

And then they could do `_build/prod/rel/my_app/bin/my_app eval "MyApp.PolynRelease.migrate"`? And they could also do things like add their module to their own supervisor tree so it tries to run on boot?

Reply:
@mmmries Are you thinking this would come up for things like umbrella apps? Do you think it would be better to go the Phoenix route and let each app generate its own `MyApp.Release` module (e.g. `mix polyn.gen.release`) or if they already have that module inject a polyn `migrate` function into it?

https://hexdocs.pm/phoenix/1.7.2/releases.html